config/initializers/redis.rb

```diff
@@ -0,0 +1,2 @@
+redis_url = URI(Figaro.env.redis_url)
+$redis = Redis.new(host: redis_url.host, port: redis_url.port)
```

Comment:
what do you usually do to config / access redis for a rails app, @dogweather ? having a hard time finding good resources